packages/stream/solana/utils.ts

```diff
@@ -35,33 +46,33 @@ export const decodeStream = (buf: Buffer): DecodedStream => {
     escrowTokens: new PublicKey(raw.escrow_tokens),
     streamflowTreasury: new PublicKey(raw.streamflow_treasury),
     streamflowTreasuryTokens: new PublicKey(raw.streamflow_treasury_tokens),
-    streamflowFeeTotal: new BN(raw.streamflow_fee_total, LE),
-    streamflowFeeWithdrawn: new BN(raw.streamflow_fee_withdrawn, LE),
-    streamflowFeePercent: new BN(raw.streamflow_fee_percent, LE),
-    partnerFeeTotal: new BN(raw.partner_fee_total, LE),
-    partnerFeeWithdrawn: new BN(raw.partner_fee_withdrawn, LE),
-    partnerFeePercent: new BN(raw.partner_fee_percent, LE),
+    streamflowFeeTotal: BigNumber(toUInt64String(raw.streamflow_fee_total)),
+    streamflowFeeWithdrawn: BigNumber(toUInt64String(raw.streamflow_fee_withdrawn)),
```

Comment:
why similar constructions were converted to different calls?

from ⬇️
```
streamflowFeeWithdrawn: new BN(raw.streamflow_fee_withdrawn, LE),
streamflowFeePercent: new BN(raw.streamflow_fee_percent, LE),
```
to ⬇️
```
streamflowFeeWithdrawn: BigNumber(toUInt64String(raw.streamflow_fee_withdrawn)),
streamflowFeePercent: BigNumber(raw.streamflow_fee_percent),
```

Reply:
very interesting question - since our fees are <1 this was always 0 (since BN can't do decimals), fortunately this doesn't seem to be used anywhere **except** in topup fee calculation in SUI client, I didn't have the time to test whether it can actually cause a bug - someone topping up a SUI stream without paying fees.